content/docs/refs-and-the-dom.md

```diff
@@ -151,11 +195,11 @@ In rare cases, you might want to have access to a child's DOM node from a parent
 
 While you could [add a ref to the child component](#adding-a-ref-to-a-class-component), this is not an ideal solution, as you would only get a component instance rather than a DOM node. Additionally, this wouldn't work with functional components.
 
-Instead, in such cases we recommend exposing a special prop on the child. The child would take a function prop with an arbitrary name (e.g. `inputRef`) and attach it to the DOM node as a `ref` attribute. This lets the parent pass its ref callback to the child's DOM node through the component in the middle.
+Instead, in such cases we recommend exposing a special prop on the child. The child would take a prop with an arbitrary name (e.g. `inputRef`) and attach it to the DOM node as a `ref` attribute. This lets the parent pass its ref to the child's DOM node through the component in the middle.
```

Comment:
Unrelated to this diff, but...

> The child would take a prop with an arbitrary name (e.g. `inputRef`)

I don't know about the use of the word "arbitrary" here. I wonder if this wouldn't read better:

> Instead, in such cases we recommend exposing a special prop on the child. This prop can be named anything other than `ref` (e.g. `inputRef`). The child component can then forward the prop to the DOM node as a `ref` attribute. This lets the parent pass its ref to the child's DOM node through the component in the middle.